src/dmd/backend/glocal.d

```diff
@@ -6,25 +6,81 @@
  *              Copyright (C) 2000-2018 by The D Language Foundation, All Rights Reserved
  * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
  * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
- * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/glocal.c, backend/glocal.c)
- * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/backend/glocal.c
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/glocal.d, backend/glocal.d)
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/backend/glocal.d
  */
 
-#if !SPP
+module backend.glocal;
```

Comment:
This should reside in the `dmd` package like all the other D files in the `backend` package.